src/Model/ImageTrait.php

```diff
@@ -66,6 +66,11 @@ public function incrementProcessingTries(int $increment = 1): void
         $this->processingTries += $increment;
     }
 
+    public function resetProcessingTries(): void
+    {
+        $this->processingTries = 0;
```

Comment:
Maybe it would be most correct to also reset the `processingRetryAt` and in this case, maybe this method should be renamed to something else, i.e. `resetProcessingRetryStatus`

Reply:
Oh, I did what we intended, but I misunderstood your change suggestion. :)

See https://github.com/Setono/SyliusImagePlugin/pull/28/commits/1df7e74ff5259b7748ec19f977ea77f3d67104bd

Reply:
Ah, I meant the method on the `ImageInterface` should do both the tries reset and the timestamp reset